include/ck/wrapper/layout.hpp

```diff
@@ -23,13 +23,19 @@ struct Layout
     static constexpr auto I0 = Number<0>{};
     static constexpr auto I1 = Number<1>{};
 
-    // Generate default idxs tuple (idx with all merged nested shapes)
+    /**
+     * \brief Generate default idxs tuple (idx with all merged nested shapes)
```

Comment:
I guess it would be best not to use any abbreviations like "idxs" in the descriptions. It does not need to be that "brief". And from only these comment it wouldn't really be clear to me what this function does.